docs/src/pages/components/selects/NativeSelects.tsx

```diff
@@ -204,14 +198,11 @@ export default function NativeSelects() {
         <FormHelperText>Required</FormHelperText>
       </FormControl>
       <FormControl variant="outlined" className={classes.formControl}>
-        <InputLabel ref={inputLabel} htmlFor="outlined-age-native-simple">
-          Age
-        </InputLabel>
+        <InputLabel htmlFor="outlined-age-native-simple">Age</InputLabel>
         <Select
           native
           value={state.age}
           onChange={handleChange('age')}
-          labelWidth={labelWidth}
```

Comment:
We need that:
![Screenshot from 2020-03-11 21-00-56](https://user-images.githubusercontent.com/12292047/76458821-6c0cfe80-63db-11ea-93ef-a3e20fbb8b13.png)

For a Select with a label you either need to pass the same node to `label` or specify `labelWidth`

Reply:
@captain-yossarian Don't forget the `label` prop, it was part of the original diff suggestion :)

Reply:
@oliviertassinari @eps1lon Sorry, I did not notice the `label` prop